toolkit/tools/specreader/specreader.go

```diff
@@ -442,12 +456,18 @@ func parseProvides(rpmsDir, srpmPath string, list []string) (providerlist []*pkg
 				return
 			}
 
+			isToolchain := schedulerutils.IsToolchainRPM(rpmPath, toolchainRPMs)
+			if isToolchain {
+				rpmPath = convertToToolchainRpmPath(rpmPath, packagearch, toolchainDir)
+			}
+
 			providerPkgVer := &pkgjson.Package{
 				Provides:     newProviderVer[0],
 				SrpmPath:     srpmPath,
 				RpmPath:      rpmPath,
 				Architecture: packagearch,
 				Requires:     reqlist,
+				IsToolchain:  isToolchain,
```

Comment:
Is this new field ever used?

Reply:
Hmm, no. I think this is left over from a previous implementation where the toolchain state of a package was passed to the grapher. Will remove.